src/content_script.js

```diff
@@ -502,7 +503,6 @@ function contentScriptInit(resetSwitch, msg) {
   // callCount = callCount + 1;
   // console.log(call, callCount);
   // console.log(source, ": ", checkForTrackers);
-  // console.log(msg);
```

Comment:
⚡️ this line in particular.

Reply:
I need to commit something to auto close the issue. 🙃